parser/parser.y

```diff
@@ -2285,12 +2286,22 @@ FunctionCallNonKeyword:
 	}
 |	"DATE_ADD" '(' Expression ',' "INTERVAL" Expression TimeUnit ')'
 	{
-		$$ = &expression.DateAdd{
+		$$ = &expression.DateArith{
+			Op:"ADD",
```

Comment:
use a const int value instead of string, like

const (
  DateAdd 
  DateSub
)